shared/productTypes.tsx

```diff
@@ -693,12 +705,14 @@ export const GROUPED_PRODUCT_TYPES: {
 		...PRODUCT_TYPES.contributions,
 		mapGroupedToSpecific: () => PRODUCT_TYPES.contributions,
 		groupFriendlyName: 'contributions',
+		groupedProductType: 'contributions',
```

Comment:
Because `GroupedProductType` extends `ProductType` we have to include the `groupedProductType` property in the grouped property types and refer back to themselves. We can't omit this property when extending `ProductType` due to the `mapGroupedToSpecific` function which may return a grouped product type instead of a product type. (Although this is a fallback that shouldn't ever happen.)